testing/test_actors/actors/fil-gas-calibration-actor/src/actor.rs

```diff
@@ -160,10 +160,26 @@ fn on_send(p: OnSendParams) -> Result<()> {
 }
 
 fn on_event(p: OnEventParams) -> Result<()> {
-    match p.mode {
-        EventCalibrationMode::Shape(_) => on_event_shape(p),
-        EventCalibrationMode::TargetSize(_) => on_event_target_size(p),
+    let mut value = vec![0; p.total_value_size];
+
+    let iterations = p.iterations as u64;
+    for i in 0..iterations {
+        random_mutations(&mut value, p.seed + i, MUTATION_COUNT);
+
+        let entries: Vec<_> = random_chunk(&value, p.entries, p.seed + i)
+            .into_iter()
+            .map(|d| Entry {
+                flags: p.flags,
+                key: char::MAX.to_string(),
```

Comment:
don't we want to vary the key here, and also test for utf8 chars so we can make sure its parsed correctly in the kernel (see my comment above)

Reply:
I'll add a comment.

I'm testing varying the number of entries and the size of the values, but _not_ the keys as that adds too many variables. Instead:

1. I'm independently testing the cost of validating utf8.
2. I'm assuming that all the _other_ costs related to keys will be the same as the costs for values (both byte buffers under the covers).